atlasdb-impl-shared/src/test/java/com/palantir/atlasdb/keyvalue/api/watch/TimestampMappingTest.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * (c) Copyright 2023 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.keyvalue.api.watch;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import com.palantir.lock.watch.LockWatchVersion;
+import com.palantir.logsafe.exceptions.SafeIllegalArgumentException;
+import com.palantir.logsafe.exceptions.SafeIllegalStateException;
+import java.util.UUID;
+import org.junit.Test;
+
+public class TimestampMappingTest {
+    @Test
+    public void single_leader_mapping_is_valid() {
```

Comment:
non-blocking, but we generally stick with camel case naming for our tests :)

(Wonder if this is something we could enforce via a lint...)

Reply:
I can update.